src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -16,17 +19,35 @@ public class CartPage {
     private final By subscription_input = By.id("susbscribe_email");
     private final By subscribeBtn_button = By.id("subscribe");
     private final By successAlert_div = By.xpath("//div[@class='alert-success alert']");
+    private By productName(String name) {
+        return By.xpath("//a[normalize-space()='" + name + "']");
+    }
+    private By productPrice(Integer ID) {
+        return By.xpath("//tr[@id=\"product-" + ID + "\"]//td[@class='cart_price']//p");
+    }
+    private By productQuantity(Integer ID) {
+        return By.xpath("//tr[@id=\"product-" + ID + "\"]//button[@class='disabled']");
+    }
+    private By productTotalPrice(Integer ID) {
+        return By.xpath("//tr[@id=\"product-" + ID + "\"]//p[@class='cart_total_price']");
+    }
```

Comment:
- Not sure to accept those locators as the user wouldn't know the ids of the items, this should be something like the name of the item, not the id
- Please follow the naming convention for the parameters variables (lowerCamaelCase).. ID should be id